pkg/manifest/manifest.go

```diff
@@ -23,6 +23,8 @@ type Interface interface {
 	Remove(string)
 	// Entry returns a manifest entry if one is found in the specified path.
 	Entry(string) (Entry, error)
+	// Length returns the amount of entries in the manifest.
```

Comment:
This is misleading. In implementations which are not the flat simple json this would be not so trivial. I would go with a general comment like `Length returns an implementation specific count of elements in the manifest.`

Reply:
done, thanks